script.js

```diff
@@ -0,0 +1,32 @@
+document.body.addEventListener('click', function(event) {
+  const ball = document.querySelector('#ball');
+  const field = document.querySelector('#field');
+  let fieldCoords = field.getBoundingClientRect();
+  console.log(+getComputedStyle(field).borderWidth.slice(0, -2));
+  let ballTop = event.clientY - fieldCoords.top - +getComputedStyle(field).borderWidth.slice(0, -2) - ball.offsetHeight/2;
```

Comment:
borderWidth.slice(0, -2) -- this is pretty bad. You can find properties which will return value in pixels. 

If you can't find them, at least create a method to extract number from a string.